src/client/oidc-react.ts

```diff
@@ -308,6 +310,45 @@ export function createOidcClient(): Client {
     ...clientFunctions
   };
   bindEvents(manager, { onAuthChange, eventTrigger, setError });
+
+  const pollFunction = async (): Promise<Response | undefined> => {
+    const uri = await manager.metadataService.getUserInfoEndpoint();
+    const tokens = getUserTokens();
+    const accessToken = tokens && tokens.accessToken;
+    if (!accessToken) {
+      return Promise.reject(new Error('Access token not set'));
+    }
+    const headers = new Headers();
+    headers.append('Authorization', `Bearer ${accessToken}`);
+
+    return fetch(uri, {
+      method: 'GET',
+      headers
+    });
+  };
+
+  const poller = createHttpPoller({
```

Comment:
The value name should better describe what it is. Perhaps something like `remoteSessionValidityPoller`.

Reply:
Renamed to "userSessionValidityPoller" :)